workspaces/spectacle/src/graphql/schema.ts

```diff
@@ -167,7 +167,7 @@ type QueryParameters {
   id: String!
 
   # Root shape ID for the QueryParameter. Look at the shapeChoices query getting more information about the root shape
-  rootShapeId: String!
+  rootShapeId: String
```

Comment:
Query parameter root shapes _can_ be null (they shouldn't but our query parameters rely on two events)